lua/cli_flash.lua

```diff
@@ -556,6 +558,7 @@ atModeArgs = {
 	list_interfaces = { mode = MODE_LIST_INTERFACES,   required_args = {},                              optional_args = {"t", "jf", "jr"}},
 	detect_netx     = { mode = MODE_DETECT_CHIPTYPE,   required_args = {},                              optional_args = {"p", "t", "jf", "jr"}}, 
 	reset_netx      = { mode = MODE_RESET,             required_args = {},                              optional_args = {"p", "t", "jf", "jr"}},
+	identify 		= { mode = MODE_IDENTIFY,		   required_args = {},								optional_args = {"p", "t", "jf", "jr"}},

```

Comment:
Please keep the indentation and spacing consistent with the environment. We're supposed to use spaces for indentation, but didn't always do so. Maybe configure the editor to display spaces and tabs.